### PR TITLE
kibana 4.3.2 kibana 4.1.5

### PR DESCRIPTION
**Pull request by jasontedor**
This pull request updates: 
 - the Kibana formula for version 4.1.5, the latest stable version of the 4.1.x line of Kibana
 - the Kibana formula for version 4.3.2, the latest stable version of the 4.3.x line of Kibana

### CODE REVIEW

File: kibana41.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Kibana41 < Formula
   desc "Analytics and search dashboard for Elasticsearch"
   homepage "https://www.elastic.co/products/kibana"
-  url "https://github.com/elastic/kibana.git", :tag => "v4.1.4", :revision => "3df681b3f205b2b75e11ddf68c0985f80460d7bc"
+  url "https://github.com/elastic/kibana.git", :tag => "v4.1.4", :revision => "04efeb4ca2305809137225ae6aa22e13de899d51"
```
Comment (DomT4) [inline kibana41.rb]: This is failing on CI I'm afraid:
```
==> Checking out tag v4.1.4
Error: v4.1.4 tag should be 04efeb4ca2305809137225ae6aa22e13de899d51
but is actually 3df681b3f205b2b75e11ddf68c0985f80460d7bc
```
It looks like you forgot to update the `:tag`?

Comment (jasontedor) [inline kibana41.rb]: Thanks @DomT4. I just force pushed a fixup.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/e969916a50a91529287d722285b83d1612aad321 & https://github.com/Homebrew/homebrew-versions/commit/be5e2c7cc56f991b67652e8bc6e2e3a35bdd0294. Thanks again @jasontedor! :)

**Comment by jasontedor** [comment]
Thanks @DomT4! :)

**Pull request closed without merging**